### PR TITLE
Update to Python 3

### PR DESCRIPTION
**Pull request by hwine**
Update all active scripts to work under python 3 (3.6 or later, tested under 3.8.3), and remove use of `.credentials` file for storing auth token in plain text.

All scripts:
- compile (`python -m compileall`)
- startup okay (see below)

No further testing was done, but no requirement versions were changed.

### CODE REVIEW

**Comment by fwenzel** [comment]
Love it - curious, have you tried pipenv yet? I recently flipped some scripts from complicated virtualenv + pip logic to pipenv. (Though I suppose if you run things in docker containers you can omit the virtualenv altogether and just install things directly inside the container).

**Comment by hwine** [comment]
Hi, Fred! Good to hear from you.

I tried `pipenv` and found it too error-prone to use. I kept invoking it in my home directory and having to clean up. I do agree that some helper is needed nowadays. 

I am undecided on packaging as docker, although I have a branch ([`r2d2`](https://github.com/hwine/github-org-scripts/tree/r2d2) in my fork) that uses [`repo2docker`](https://repo2docker.readthedocs.io/en/latest/index.html) successfully (so I don't need py2 installed on my system anymore).

File: README.md [hunk 1]
```diff
@@ -60,20 +57,13 @@ feature. Use the ``--help`` option for more information.
...
-Yes, you have to go to the next page and re-run this to get another file.
-
 #### hooks.py
 Analyzes a list of audit log export files (from the JS script) for hook/service creation/deletion and provides a summary. Use it to show commonly used apps/services/webhooks across the org.
 
 ### old_repos.py
 Generate a list of empty (should be deleted) repositories as well as untouched repos (might need to be archived).
 
 ## License
-This code is free software and licensed under an MPL-2.0 license. &copy; 2015-2018 Fred Wenzel and others. For more information read the file ``LICENSE``.
```
Comment (fwenzel) [inline README.md:77]: This is flattering, though I think by now you deserve waaaaay more credit, @hwine !

**Review by fwenzel — COMMENTED** [review]
Interesting, thank you for sharing. I will admit that pipenv is a bit... magical, so probably hard to pinpoint when something breaks (ugh)

For scripts like these, agree that spinning up whole docker instances every time seems too heavy-handed.

Hope you're well!

**Pull request merged**